src/index.js

```diff
@@ -0,0 +1,150 @@
+const state = {
+    temp: 70,
+    city: 'Seattle',
+};
+
+const tempInfo = document.getElementById('tempInfo');
+const tempUp = document.getElementById('tempUp');
+const tempDown = document.getElementById('tempDown');
+const landScape = document.getElementById("landScape");
+const cityInput = document.getElementById('cityInput');
+const cityValue = document.getElementById('cityInput').value;
+const currentCity = document.getElementById('currentCity');
+const realTemp = document.getElementById('getTemp');
+const skySelect = document.getElementById('skies');
+const sky = document.getElementById('sky').value;
+const resetCity = document.getElementById('resetCity')
+
+const updateWeather = () => {
+    tempInfo.textContent = state.temp
+};
+
+const increaseWeather = () => {
+    state.temp += 1;
+    updateWeather();
+    updatelandScape();
+};
+
+const decreaseWeather = () => {
+    state.temp -= 1;
+    updateWeather();
+    updatelandScape();
+};
+
+const updatecityInput = () => {
+    const updatedCity = document.getElementById('cityInput').value;
+    const currentCity = document.getElementById('currentCity');
+    state.city = updatedCity;
+    currentCity.textContent = state.city;
+};
+
+const updateresetCity = () => {
+    const updatedCity = document.getElementById('cityInput');
+    const currentCity = document.getElementById('currentCity');
+    state.city = "Seattle";
+    currentCity.textContent = state.city;
+    updatedCity.value = state.city;
+};
+
+const updateSky = () => {
+    const sky = document.getElementById('sky');
+    const skyValue = document.getElementById('skies').value;
+    if (skyValue === 'sunny') {
+        sky.textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️'
+    } else if (skyValue === 'cloudy') {
+        sky.textContent = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️'
+    } else if (skyValue === 'rainy') {
+        sky.textContent = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧'
+    } else if (skyValue === 'snowy') {
+        sky.textContent = '❄️☃️❄️☃️❄️☃️❄️☃️❄️☃️❄️☃️❄️'
+    }
+};
+
+
+const updatelandScape = () => {
+    const sky = document.getElementById('sky');
+    if (state.temp > 80) {
+        tempInfo.style.color = 'red'
+        sky.textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️'
+        landScape.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂'
+    } else if (state.temp > 70) {
+        tempInfo.style.color = 'orange'
+        sky.textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️'
+        landScape.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷'
+    } else if (state.temp > 60) {
+        tempInfo.style.color = 'yellow'
+        sky.textContent = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️'
+        landScape.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃'
+    } else if (state.temp > 50) {
+        tempInfo.style.color = 'green'
+        sky.textContent = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧'
+        landScape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲'
+    } else if(state.temp < 49) {
+        tempInfo.style.color = 'teal'
+        landScape.textContent = '❄️☃️❄️☃️❄️☃️❄️☃️❄️☃️❄️☃️❄️'
+    }
```

Comment:
Check out the logic here, see how similar the branches are of the conditional statement. We look for whether the temperature is within a range, and pick a color accordingly. What if we had a list of records that we could iterate through to find the values. We could set up something like

```javascript
const COLORS_AND_LANDSCAPES = [ 
  { upperBound: 49, color: "teal", background: '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂'}, 
  { upperBound: 59, color: "green", background: '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷'}, 
  { upperBound: 69, color: "yellow", background: '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃'}
];

Then your method would iterate through COLORS_AND_LANDSCAPES and find the first record that has an upper bound higher than our temperature, then use it as the source of picking the color which would help make the function a bit more concise.

This could accommodate a scenario where you might have even more colors/landscapes in the future, which would prevent your conditional statement from being a really long block of if/elif/elif and so on